android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -98,6 +98,11 @@ public int getAppVersion() {
     }
 
     @NonNull
+    public String getAppVersionName() {
+        return applicationContext.getResources().getString(R.string.osb_app_version_name);
```

Comment:
@DwayneJengSage do you know the format of AppVersionName expected in a Bridge data archive's info.json file?